### PR TITLE
Moved to common step to beforeEach scope within the test.describe group and url update to POM comparision image

### PR DESCRIPTION
**Pull request by vidyavemulapalli**
…

### CODE REVIEW

File: docs/POMComparision.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This section provides a comparison between Traditional POM and Preferred POM in
 
 The following image illustrates the differences between Traditional and Preferred POM in Spec files:
 
-![Traditional vs. Preferred POM Spec file](https://github.com/vasu31dev/playwright-ts-template/blob/main/docs/images/SpecFile-Comparision.png?raw=true)
+![Traditional vs. Preferred POM Spec file](https://raw.githubusercontent.com/vasu31dev/playwright-ts-template/main/docs/images/SpecFile-Comparision.png)
```
Comment (vidyavemulapalli) [inline docs/POMComparision.md:9]: I need to test this url once it gets merged. I am unable to verify it from Readme preview

File: README.md [hunk 2]
```diff
@@ -267,32 +267,38 @@ import * as LoginPage from 'tests/pages/preferredPOM/sauce-demo-login-page';
 import * as MiniCart from 'tests/pages/preferredPOM/sauce-demo-mini-cart';
 import * as ProductsPage from 'tests/pages/preferredPOM/sauce-demo-products-page';
 
-test.describe('Saucedemo tests for successful, unsuccessful logins and add product to cart', () => {
+test.describe.configure({ mode: 'parallel' });
+
+// beforEach hook in scope with entire spec file
+// eslint-disable-next-line playwright/require-top-level-describe
```
Comment (vidyavemulapalli) [inline README.md:273]: @vasu31dev I have to disable this eslint check in order to add beforeEach hook at spec file level

File: tests/specs/sauce-demo-all-pass.spec.ts [hunk 3]
```diff
@@ -10,31 +10,34 @@ import * as ProductsPage from '../pages/sauce-demo/sauce-demo-products-page';
 */
 test.describe.configure({ mode: 'parallel' });
 
-test.describe('Saucedemo tests for successful, unsuccessful logins and add product to cart @smoke', () => {
-  //navigating to sauce demo home page before each test
-  test.beforeEach('Navigating to sauce demo page', async () => {
-    await LoginPage.navigateToSauceDemoLoginPage();
-  });
+// beforEach hook in scope within entire spec file
+// eslint-disable-next-line playwright/require-top-level-describe
```
Comment (vidyavemulapalli) [inline tests/specs/sauce-demo-all-pass.spec.ts:14]: @vasu31dev i have to disable this eslint check for this line in order to add beforeEach hook at spec file level

File: tests/specs/sauce-demo-all-pass.spec.ts [hunk 4]
```diff
@@ -10,29 +10,34 @@ import * as ProductsPage from '../pages/sauce-demo/sauce-demo-products-page';
 */
 test.describe.configure({ mode: 'parallel' });
 
-test.describe('Saucedemo tests for successful, unsuccessful logins and add product to cart @smoke', () => {
+// beforEach hook in scope within entire spec file
+// eslint-disable-next-line playwright/require-top-level-describe
+test.beforeEach('Navigating to sauce demo page', async () => {
+  await LoginPage.navigateToSauceDemoLoginPage();
+});
+
```
Comment (vasu31dev) [inline tests/specs/sauce-demo-all-pass.spec.ts:18]: Add test. beforeEach inside the test .describe

**Review by vasu31dev — CHANGES_REQUESTED** [review]

File: tests/specs/sauce-demo-all-pass.spec.ts (hunk 4 above)
Comment (vidyavemulapalli) [reply tests/specs/sauce-demo-all-pass.spec.ts:18]: Deleted the beforeEach hook in scope with spec file and added it with in the scope of test.describe block

**Review by vasu31dev — APPROVED** [review]

**Pull request merged**